### ISSUE TITLE
After Login Successful Crass App

### ISSUE THREAD

**Issue by opmaurya1995** [Odoo-mobile/framework#364]
I am try to connect my local database odoo community version 10 on mobile app, after successful login it will crashed and not open. please check attached file.
![screenshot from 2018-01-27 22-04-10](https://user-images.githubusercontent.com/13413423/35474202-0eb3087e-03b1-11e8-8a64-111786d535b2.png)

**Comment by ghost** [comment]
I am also experiencing this issue on my Samsung Galaxy S2 Tablet (JellyBean, 4.2.2)

**Comment by naitikvithlani** [comment]
@opmauryasarthi @lramage94 Can you please add log report of application crashed.

**Comment by ghost** [comment]
@naitikvithlani I have a user using the release apk included in the repository that is having issues. I am not sure how to get a log from him. Any ideas on that?

**Comment by naitikvithlani** [comment]
you can get full source code from git and run it in android studio,
You can customise it as per your need.

**Comment by ghost** [comment]
@naitikvithlani yes, I did that. However, I have a user that I was trying to have demo the software and it crashed for him. I am not in the same physical location and he is not a developer, he's the end user. How would I be able to get a log from his device?

**Comment by kasim1011** [comment]
@lramage94 if you've signed app without updating its dependencies then you might require to update `Google Play Service` dependency at [build.gradle](https://github.com/Odoo-mobile/framework/blob/master/app/build.gradle) from `compile 'com.google.android.gms:play-services-gcm:10.0.1'` to `compile 'com.google.android.gms:play-services-gcm:11.8.0'` and you need add **Google Maven** dependency at [build.gradle](https://github.com/Odoo-mobile/framework/blob/master/build.gradle) like [this](https://developer.android.com/studio/build/dependencies.html#google-maven)

    allprojects {
        repositories {
            google()

            // If you're using a version of Gradle lower than 4.1, you must instead use:
            // maven {
            //     url 'https://maven.google.com'
            // }
            // An alternative URL is 'https://dl.google.com/dl/android/maven2/'
        }
    }
